bookkeeper-server/src/main/java/org/apache/bookkeeper/util/collections/ConcurrentLongHashMap.java

```diff
@@ -382,6 +382,9 @@ V get(long key, int keyHash) {
         }
 
         V put(long key, V value, int keyHash, boolean onlyIfAbsent, LongFunction<V> valueProvider) {
+            if (value == null) {
```

Comment:
How about moving the check to the `computeIfAbsent`? I saw it checked whether the provider is null or not, we should also check the value.

Reply:
Yes, this modification is also correct. maybe I can move to the `computeIfAbsent`

Reply:
But I feel that it is more appropriate to do the verification in the put function, because this problem exists in ```org.apache.bookkeeper.util.collections.ConcurrentLongHashMap.Section#put``` , not the caller ```ConcurrentLongHashMap#computeIfAbsent```